cmd/minikube/cmd/ssh.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/docker/machine/libmachine"
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"k8s.io/minikube/pkg/minikube/cluster"
+	"k8s.io/minikube/pkg/minikube/constants"
+)
+
+// dockerSSHCmd represents the docker-ssh command
+var dockerSSHCmd = &cobra.Command{
```

Comment:
Sorry about that, I've changed the message and the function name.  Both the
ssh command and the associated test appear to be working but I was
wondering if I could get some feedback on my changes to ssh_mock.go as I
believe there is more work to be done regarding closing the session for the
mock as currently the SSHServer.Session bool can only be set to true and is
not set back to false when the session closes.  I know that you addressed
this in your original message but exactly how to do it is unclear to me.
Thanks!

On Mon, May 30, 2016 at 10:04 PM, dlorenc <notifications@github.com> wrote:

> In cmd/minikube/cmd/ssh.go
> <https://github.com/kubernetes/minikube/pull/134#discussion_r65124054>:
>
> > +*/
> > +
> > +package cmd
> > +
> > +import (
> > +	"os"
> > +
> > +	"github.com/docker/machine/libmachine"
> > +	"github.com/golang/glog"
> > +	"github.com/spf13/cobra"
> > +	"k8s.io/minikube/pkg/minikube/cluster"
> > +	"k8s.io/minikube/pkg/minikube/constants"
> > +)
> > +
> > +// dockerSSHCmd represents the docker-ssh command
> > +var dockerSSHCmd = &cobra.Command{
>
> Rename this to sshCmd as well?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/minikube/pull/134/files/bc3719f6d26b317e5adb97b13cfeeac8f8becfdd#r65124054>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB3Hql8pCmnZOKzpjll0HgFR6dOyL4ewks5qG8FngaJpZM4Ip9OY>
> .
>